extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -294,12 +295,12 @@ export class GmailElementReplacer implements WebmailElementReplacer {
       }
     }
     // conversation top-right icon buttons
-    if (convoUpperIcons.length) {
+    if (convoUpperIconsContainer.length) {
       if (useEncryptionInThisConvo) {
-        if (!convoUpperIcons.is('.appended') || convoUpperIcons.find('.use_secure_reply').length) {
-          // either not appended, or appended icon is outdated (convo switched to encrypted)
-          this.addfcConvoIcon(convoUpperIcons, this.factory.btnWithoutFc(), '.show_original_conversation', () => {
-            convoUpperIcons.find('.gZ').trigger('click');
+        if (!convoUpperIconsContainer.is('.appended') || convoUpperIconsContainer.find(convoUpperIcons).length) {
+          this.addfcConvoIcon(convoUpperIconsContainer, this.factory.btnWithoutFc(), '.show_original_conversation', () => {
+            const openOriginalInNewTabIndex = convoUpperIcons.length - 1; // the open in new tab icon button has always been in the last item
+            convoUpperIconsContainer.find(convoUpperIcons).eq(openOriginalInNewTabIndex).trigger('click');
```

Comment:
probably we can just use just `.find(convoUpperIcons).last()` instead of calculating index

Reply:
Thank you, Roma. Will apply this instead.